packages/ds-healthcare-gov/src/components/Header/Menu.tsx

```diff
@@ -24,12 +24,17 @@ const Menu = (props: MenuProps) => {
   });
 
   return (
-    <div id="hc-c-menu" hidden={!props.open} className={classes}>
+    <nav
+      aria-label="Profile and application"
+      id="hc-c-menu"
+      hidden={!props.open}
+      className={classes}
+    >
```

Comment:
This actually seems like we're reversing what you did [in the previous nav cleanup changes](packages/ds-healthcare-gov/src/components/Header/Menu.tsx). Is there nuance that I'm missing here?

Reply:
DK specified this should be a `<nav>` element in the Jira ticket, so it seems the `<div>` was an incorrect change previously.

Reply:
Okay, maybe we should ask @SparrowSpit to take a look at this version and give a final ruling

Reply:
I can't seem to view the link Patrick pasted for the old nav cleanup changes. Was there a reason to use the div ? It would seem the nav element would semantically be correct but I just want to make sure there isn't some historical context I'm missing.

Reply:
Updated the link to point to the previous change commit. Here's [the pull request](https://github.com/CMSgov/design-system/pull/1854), which also has a link to [its Jira ticket](https://jira.cms.gov/browse/WNMGDS-1402)

Reply:
That's helpful. Thank you. So, it would seem to me that we are going back to having the issue DK described where we have nested nav elements.

Reply:
Yes. Is that an issue if the nested navs have labels? Or is it best to revert back to the div solution and omit the aria-label?

Reply:
Ok. Sarah and I chatted about this to help me understand the historical context. Let's do this:

- stick with the div on what is now the nested nav element
- keep the outer nav element that contains both the user name and the menu+actions
- use the aria-label="Profile, applications, and coverage" on the outer nav